root/programs/C#/Frameworks/Infrastructure/Business/RichClient/Business/MyBaseLogic2CS.cs

```diff
@@ -157,7 +157,7 @@ public abstract class MyBaseLogic2CS : BaseLogic2CS
                 else if (parameterValue.ActionType.Split('%')[0] == "ODP")
```

Comment:
@daisukenishino 

I think that `MyBaseLogic2CS` class is present only to maintain backward compatibility.
*Open Touryo version 2.0* is released. This version is major version up.
So, it might be a good idea to remove classes that is present only to maintain backward compatibility.